doc/user/flycheck-versus-flymake.rst

```diff
@@ -315,33 +213,33 @@ to set this variable interactively.
 Multiple syntax checkers per buffer
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-**Flymake** can only use a single syntax checker per buffer.  Effectively, the
-user can only use a single tool to check a buffer, for instance either PHP Mess
-Detector or PHP CheckStyle.  Third party extensions to Flycheck work around this
-limitation by supplying custom shell scripts to call multiple syntax checking
-tools at once.
+**Flymake** will use all the backends added to the
+`flymake-diagnostic-functions` hook to check a buffer; all backends are started
+at the same time, but errors are reported in the buffer as soon as a backend
+returns them.  Also, backends can be written to report errors for the visible
+region of the buffer first, and collect errors for hidden regions later.
 
-**Flycheck** can easily apply multiple syntax checkers per buffer.  For
-instance, Flycheck will check PHP files with PHP CLI first to find syntax
-errors, then with PHP MessDetector to additionally find idiomatic and semantic
-errors, and eventually with PHP CheckStyle to find stylistic errors.  The user
-will see all errors reported by all of these utilities in the buffer.
+**Flycheck** can also apply multiple syntax checkers per buffer, but in sequence
+rather than concurrently.  For instance, Flycheck will check PHP files with PHP
+CLI first to find syntax errors, then with PHP MessDetector to additionally find
+idiomatic and semantic errors, and eventually with PHP CheckStyle to find
+stylistic errors.  The user will see all errors reported by all of these tools
+in the buffer.  However, if the first checker reported at least one error, then
+the subsequent checkers would not be run.
 
 Errors
 ------
 
 Error levels
 ~~~~~~~~~~~~
 
-**Flymake** supports error and warning messages.  The pattern of warning
-messages is *hard-coded* in Emacs 24.3, and only became customizable in Emacs
-24.4.  The patterns to parse messages are kept separate from the actual syntax
-checker.
+**Flymake** supports error, warning and note levels by default.  Additional
+error levels can easily be defined by adding them to
+`flymake-diagnostic-types-alist`.
 
-**Flycheck** supports error, warning and info messages.  The patterns to parse
-messages of different levels are part of the syntax checker definition, and thus
-specific to each syntax checker.  Flycheck allows to define new error levels for
-use in custom syntax checkers with `flycheck-define-error-level`.
+**Flycheck** supports error, warning and info messages.  Flycheck allows you to
+define new error levels for use in custom syntax checkers with
+`flycheck-define-error-level`.
```

Comment:
I'd remove this section, since they both work similarly.

Reply:
Done.